mixer/adapter/servicecontrol/handler.go

```diff
@@ -117,13 +117,32 @@ func (h *handler) HandleApiKey(ctx context.Context, instance *apikey.Instance) (
 
 // HandleServicecontrolReport handles reporting metrics and logs.
 func (h *handler) HandleServicecontrolReport(ctx context.Context, instances []*servicecontrolreport.Instance) error {
```

Comment:
Have you validated that this works with manual testing?  Last I knew, this adapter had fallen into a state of disrepair. It'd be nice to have some confirmation that it is workable before worrying abut improving it.